src/cswinrt/code_writers.h

```diff
@@ -8450,7 +8452,13 @@ global::System.Collections.Concurrent.ConcurrentDictionary<RuntimeTypeHandle, ob
                     }
                 }),
             bind<write_class_members>(type, false, false),
-            bind<write_custom_query_interface_impl>(type));
+            bind<write_custom_query_interface_impl>(type),
+            settings.netstandard_compat ? "" : w.write_temp(R"(
+internal sealed class %RcwFactoryAttribute : WinRTImplementationTypeRcwFactoryAttribute
```

Comment:
From what I see, this is going inside of the RCW class.  Can we make this private?

Reply:
It's not nested in the RCW, it's at the same level, so it has to be internal. I was following the same pattern of the `[WinRTExposedTypeDetails]` attributes, which are also generated as internal attributes. Since it's all just in the generated projections, it should mostly be the same? What do you think?

Reply:
The only difference is the `WinRTExposedTypeDetails` are generated in the ABI namespace.  If this is at the same level, can we also generate this class in the ABI namespace instead similar to the WinRTExposedTypeDetails?

Reply:
Sure thing! Done in https://github.com/microsoft/CsWinRT/pull/1505/commits/2d0654ee8f390b91ae8578c46afce6994488ae00 🙂